spec/facility_spec.rb

```diff
@@ -23,4 +28,208 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+  
+  describe 'can add a service' do
+    it 'can add a service' do
+      @facility.add_service('Vehicle Registration')
+      expect(@facility.services).to eq(["Vehicle Registration"])
+    end
+  end
+  
+  describe 'has registered_vehicles' do
+    it 'has registered_vehicles array' do
+      @facility.registered_vehicles
+      expect(@facility.registered_vehicles).to eq([])
+    end
+  end
+
+  describe '#collected_fees' do
+    it 'has collected_fees' do
+      @facility.collected_fees
```

Comment:
Again this is what you are calling on line 49. You will need to call 
```ruby
@facility.register_vehicle(@cruz)
```
In order to see that `collected_fees` is changing